rest-archetype/src/main/resources/archetype-resources/rest/src/main/java/HelloResource.java

```diff
@@ -0,0 +1,16 @@
+package ${package};
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("hello")
```

Comment:
There is also an HelloResource in this rest-archetype. That's why I don't see why it fails.